docs/GlobalAPI.md

```diff
@@ -19,7 +19,13 @@ import TOCInline from '@theme/TOCInline';
 
 Runs a function after all the tests in this file have completed. If the function returns a promise or is a generator, Jest waits for that promise to resolve before continuing.
 
-Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting. _Note: The default timeout is 5 seconds._
+Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting.
+
+:::note
+
+The default timeout is 5 seconds.
+
+:::
```

Comment:
Hm.. Perhaps all timeout notes should stay inline? In this case separate block feels unnecessary, they create to much visual noise.

```suggestion
Optionally, you can provide a `timeout` (in milliseconds) for specifying how long to wait before aborting. The default timeout is 5 seconds.
```